src/api/app/components/notification_excerpt_component.rb

```diff
@@ -18,11 +18,9 @@ def call
              @notifiable.reason
            when 'WorkflowRun'
              "In repository #{@notifiable.repository_full_name}"
-           else
-             ''
            end
 
-    tag.p(truncate_to_first_new_line(text), class: %w[mt-3 mb-0])
+    tag.p(truncate_to_first_new_line(text || ''), class: %w[mt-3 mb-0])
```

Comment:
Why don't you do it like it's done for `BsRequest.description`?

Reply:
Because I'm as blind as a mole. Let me fix it.

Reply:
![mole](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExYWlnenlwenE5eGE2eHlnNW5jbGF1anBsZ2Q5NGd3bmQxdGkweG4weCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/9HWPheLPma1vYrWMYY/giphy.gif)